core/src/main/java/org/apache/accumulo/core/metadata/MetadataLocationObtainer.java

```diff
@@ -115,9 +115,9 @@ public TabletLocations lookupTablet(ClientContext context, TabletLocation src, T
         range = new Range(results.lastKey().followingKey(PartialKey.ROW_COLFAM_COLQUAL_COLVIS_TIME),
             true, new Key(stopRow).followingKey(PartialKey.ROW), false);
         encodedResults.clear();
-        more = ThriftScanner.getBatchFromServer(context, range, src.tablet_extent,
-            src.tablet_location, encodedResults, locCols, serverSideIteratorList,
-            serverSideIteratorOptions, Constants.SCAN_BATCH_SIZE, Authorizations.EMPTY, 0L, null);
+        ThriftScanner.getBatchFromServer(context, range, src.tablet_extent, src.tablet_location,
+            encodedResults, locCols, serverSideIteratorList, serverSideIteratorOptions,
+            Constants.SCAN_BATCH_SIZE, Authorizations.EMPTY, 0L, null);
```

Comment:
I don't recall at the moment - but I likely just took the IDE suggested change.  If you believe this is incorrect, please feel to revert this change and we can revisit it in isolation.

Reply:
Getting rid of the extra assignment is a valid IDE suggestion, since it's never used again. However, it looks like it was supposed to be used. There may yet still be a bug here, but the code is not very well documented or intuitive. I created #3072 to investigate this.